### PR TITLE
Add System76 launch_lite_1

### PR DESCRIPTION
**Pull request by jackpot51**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [x] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/system76-launch/fu-system76-launch-device.c [hunk 1]
```diff
@@ -138,11 +138,22 @@ fu_system76_launch_device_detach(FuDevice *device, FuProgress *progress, GError
...
 	if (fu_device_get_update_message(device) == NULL) {
+		GUsbDevice *usb_device = fu_usb_device_get_dev(FU_USB_DEVICE(device));
 		g_autofree gchar *msg = NULL;
+		const gchar *unlock_keys;
+		switch (g_usb_device_get_pid(usb_device)) {
+		case 0x0001: /* launch_1 */
+			unlock_keys = "Fn+Esc";
+			break;
+		default:
+			unlock_keys = "Left Ctrl+Right Ctrl+Esc";
```
Comment (jackpot51) [inline plugins/system76-launch/fu-system76-launch-device.c:149]: All future Launch keyboards will use this combo, that is why this is the default.

File: plugins/system76-launch/fu-system76-launch-device.c [hunk 2]
```diff
@@ -138,11 +138,22 @@ fu_system76_launch_device_detach(FuDevice *device, FuProgress *progress, GError
 
 	/* generate a message if not already set */
 	if (fu_device_get_update_message(device) == NULL) {
+		GUsbDevice *usb_device = fu_usb_device_get_dev(FU_USB_DEVICE(device));
 		g_autofree gchar *msg = NULL;
+		const gchar *unlock_keys;
+		switch (g_usb_device_get_pid(usb_device)) {
```
Comment (hughsie) [inline plugins/system76-launch/fu-system76-launch-device.c:144]: you can just do `fu_usb_device_get_pid(FU_USB_DEVICE(device))` here

File: plugins/system76-launch/system76-launch.quirk [hunk 3]
```diff
@@ -1,4 +1,9 @@
-# System76 Launch
+# System76 launch_1
 [USB\VID_3384&PID_0001&REV_0001]
 Plugin = system76_launch
 CounterpartGuid = USB\VID_03EB&PID_2FF4
+
+# System76 launch_lite_1
+[USB\VID_3384&PID_0005&REV_0001]
+Plugin = system76_launch
+CounterpartGuid = USB\VID_03EB&PID_2FF9
```
Comment (jackpot51) [inline plugins/system76-launch/system76-launch.quirk:9]: Launch Lite uses the AT90USB646, which has a different bootloader PID.

Comment (hughsie) [reply plugins/system76-launch/system76-launch.quirk:9]: Ahh, so you'll likely need the plugins/dfu/dfu.quirk addition too?

**Review by hughsie — APPROVED** [review]

**Comment by jackpot51** [comment]
Thanks @hughsie!

**Pull request merged**